docs/x509/reference.rst

```diff
@@ -2099,6 +2099,33 @@ X.509 Extensions
         Returns :attr:`~cryptography.x509.oid.ExtensionOID.PRECERT_POISON`.
 
 
+.. class:: SignedCertificateTimestamps(scts)
+
+    .. versionadded:: 3.0
+
+    This extension contains
+    :class:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp`
+    instances. These can be used to verify that the certificate is included
+    in a public Certificate Transparency log. This extension is typically found
+    in OCSP responses. For SCTs in an X.509 certificate see
```

Comment:
Typically, or only?

Reply:
I actually don't know. I wrote typically as a weasel word 😄 Is it solely in OCSP responses?